pages/sell/[classId]/[nftId].vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div>
+    <h1>List NFT for sell</h1>
+    <table>
+      <thead>
+        <tr>
+          <th>#</th>
+          <th>NFT information</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>#0</td>
+          <td>
+            <div>
+              <h4>{{ nftData.data?.metadata.name }}</h4>
+              <img v-if="nftData.data?.metadata?.image" :src="convertImageSrc(nftData.data?.metadata.image)" width="256"
+                height="256" /><br />
+              Class ID:
+              <NftLink :class-id="classId" /><br />
+              NFT ID:
+              <NftLink :class-id="classId" :nft-id="nftId" />
+            </div>
+            <button @click="viewClassListings(classId)">View current listing for class</button>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+    <form @submit.prevent="createNFTListing">
+      <label for="listing-price">Price:</label>
+      <input type="number" id="listing-price" v-model="listingPrice" /> LIKE
+      <br />
+      <label for="listing-expiration">Offer expiration:</label>
+      <input type="date" id="listing-expiration" v-model="listingExpiration" :max="maxExpirationValue"
+        :min="minExpirationValue" />
+      <br />
+      <input type="submit" value="Confirm" />
+    </form>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { storeToRefs } from 'pinia';
+import { useWalletStore } from '~/stores/wallet';
+import { queryNFTClass, queryNFT, signCreateNFTListing } from '../../../utils/cosmos';
+
+const router = useRouter();
+const route = useRoute();
+const store = useWalletStore();
+const { wallet, signer } = storeToRefs(store);
+const classData = ref({} as any);
+const nftData = ref({} as any);
+const listingPrice = ref(32);
+const listingExpiration = ref(new Date(Date.now() + 15552000000).toISOString().split('T')[0]);
+
+const maxExpirationValue = computed(() => new Date(Date.now() + 15552000000).toISOString().split('T')[0]);
+const minExpirationValue = computed(() => new Date(Date.now()).toISOString().split('T')[0]);
```

Comment:
I think `computed` doesn't work with `Date.now()` since the function is not reactive?

Reply:
Yes seems nothing is changing anyway, just data would do